### ISSUE TITLE
Error with syncing channel updates

### ISSUE THREAD

**Issue by stared** [willcrichton/flowistry#20]
# Problem

Right after installation, I got the following error:

# Logs

**OS:** macOS (10.15.7)
**VSCode:** 1.60.2
**Rust:** 1.54.0 (a178d0322 2021-07-26)
**Error message**
```
info: syncing channel updates for 'nightly-2021-09-23-x86_64-apple-darwin'
info: latest update on 2021-09-23, rust version 1.57.0-nightly (308dffd25 2021-09-22)
error: component 'rust-src,rustc-dev,llvm-tools-preview' for target 'x86_64-apple-darwin' is unavailable for download for channel nightly-2021-09-23
Sometimes not all components are available in any given nightly.
```

**Full log:** https://paste.rs/tPe

**Comment by willcrichton** [comment]
Hi @stared thanks for the report. I pushed a potential fix in 0.3.18. Can you try reinstalling in ≥1 hr?

**Comment by stared** [comment]
No installation problems this time (no error).

However, when I use it (even after reload), it does not work on a more complicated codebase (everything returns null results) - tough this is a different issue and I will address it separately.
